ci/test-checks.sh

```diff
@@ -17,7 +17,7 @@ scripts/increment-cargo-version.sh check
   _ scripts/cargo-for-all-lock-files.sh tree >/dev/null
   set +e
   if ! _ git diff --exit-code; then
-    echo -e "\nError: Uncommitted Cargo.lock changes" 1>&2
+    echo -e "\nError: Uncommitted Cargo.lock changes. Run './scripts/cargo-for-all-lock-files.sh tree' and commit the result." 1>&2
```

Comment:
this line's getting kinda long. let's switch to `cat`

```suggestion
    cat <<EOF 1>&2

Error: Uncommitted Cargo.lock changes.
Run './scripts/cargo-for-all-lock-files.sh tree' and commit the result.
EOF
```